### DISCUSSION TITLE
The Adguard app 4.0 on S21 Ultra is using too much memory!

### DISCUSSION THREAD

**Discussion by dewittinfl** [AdguardTeam/AdGuardHome#5925] [Q&A]
![Screenshot_20230622_154619_Settings](https://github.com/AdguardTeam/AdGuardHome/assets/114003736/55bb830b-4e0f-471b-8923-e251555e3124)
![Screenshot_20230622_165922_Settings.jpg](https://github.com/AdguardTeam/AdGuardHome/assets/114003736/4a02a6cd-6f09-4025-a9b9-9dde68d4e5a1)

**Comment by ainar-g** [comment]
This is a repo for AdGuard Home filtering DNS server, not the AdGuard app.
